boss-ws/src/main/java/org/genomebridge/boss/http/service/BossAPI.java

```diff
@@ -32,31 +33,13 @@ public ErrorDesc( Response.Status status, String message ) {
     public ErrorDesc insertObject(ObjectDesc desc, String userName);
     public ErrorDesc updateObject(ObjectDesc desc, String objectId, String userName);
     public ErrorDesc deleteObject(String objectId, String userName);
-
-    public static class ResolveRequest {
-        public Integer validityPeriodSeconds;
-        public String httpMethod;
-        public String contentType;
-        public String contentMD5Hex;
-    }
-
-    @JsonInclude(Include.NON_NULL)
-    public static class ResolveResponse {
-        public URI objectUrl;
-        public Integer validityPeriodSeconds;
-        public String contentType;
-        public String contentMD5Hex;
-    }
-
     public ErrorDesc resolveObject(String objectId, String userName, ResolveRequest req, ResolveResponse resp);
     public ErrorDesc getResumableUploadURL(String objectId, String userName, CopyResponse resp);
 
-    public static class CopyRequest {
-        public Integer validityPeriodSeconds;
-        public String locationToCopy; // expecting something of the form "/bucket/key"
-    }
 
+    @ApiModel("Copy Response")
     public static class CopyResponse {
+        @ApiModelProperty(value=" is a pre-signed url generated to access the object for the desired operation",dataType = "URI")
```

Comment:
Language: "A pre-signed url generated to access the object for the desired operation."